### DISCUSSION TITLE
Replace Twitter API with Discord API

### DISCUSSION THREAD

**Discussion by AaronSadlerUK** [H5YR/High5YouRock#20] [Ideas]
Based off the idea by Paul Marden on Twitter we could replace the Twitter API with the Discord API.

https://twitter.com/paulmarden/status/1670077662985965576

A quick look on the Discord API shows how to return messages, but it does not show how to filter these messages:
https://discord.com/developers/docs/resources/channel#get-channel-messages

Another option would be to listen for message webhooks, and then store the last X number of message IDs which match a criteria to be displayed on the site:
https://discord.com/developers/docs/resources/webhook#create-webhook

**Comment by OwainWilliams** [comment]
A suggestion from Janae on Twitter is to make it a discord bot command like !h5yr or /h5yr so that people can opt in. 

Plus if you use #h5yr I think it links to a channel so a command may work better.

**Comment by OwainWilliams** [comment]
I'm going to give this a go as always fancied setting up a bot :)
